### PR TITLE
Work №3

### PR DESCRIPTION
**Pull request by NovikVeronika**
https://novikveronika.github.io/itirod/

I tried to do it nice, but i hope you like it. I also have a GiveUpPage, but you cannot access it due to absence of any elements with according actions. 

Warning: not suitable for mobile version (you can try it yourself but don't be angry with me after it.....))))
Can you tell me which functions i should implement for the 4th work :)
Thanks for your patience! :)

### CODE REVIEW

File: html/StartPlantingPage.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <div class="start-plant-image" width="100%">
+        <img src="../images/example-image.png" alt="" width="350px"; height="350px";>
+        <p class="start-label">Don't waste your time! Do it now!</p>
+        <p class="change-time">
+            <label for="time">Change time: </label><br>
+            <input type="time" id="time" name="time"/>
+        </p>
+    </div>
+    <aside>
+        <button class="rbutton" onclick="window.location.href='StartPage.html'">Start</button>
```
Comment (katetushkan) [inline html/StartPlantingPage.html]: I think u don't need href here) Should it start the timer?

File: html/StartPlantingPage.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <link rel="stylesheet" type="text/css" href="../css/siteStyle.css"/>
+    <link rel="stylesheet" type="text/css" href="../css/StartPlantingPage.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;400;600&display=swap" rel="stylesheet">
+</head>
+<body>
+<header>
+    <div class="text-header">
+        <label class="for-label">FOR</label>
```
Comment (katetushkan) [inline html/StartPlantingPage.html]: Should be '<p>' or `<small>`. `<label>` could be only used with the `<input>`

File: html/SignUpPage.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+</head>
+<body>
+<header>
+    <div class="text-header">
+        <label class="for-label">FOR</label>
+        <label class="rest-label">rest</label>
+    </div>
+</header>
+<form class="log-form" action="LoginPage.html">
+    <p>USER NAME</p>
```
Comment (katetushkan) [inline html/SignUpPage.html]: Only `<label>` could be used with the `<input>`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+</head>
+<body>
+<header>
+    <div class="text-header">
+        <label class="for-label">FOR</label>
+        <label class="rest-label">rest</label>
+    </div>
+</header>
+<form class="log-form" action="html/LoggedPage.html">
+    <p>USER NAME</p>
```
Comment (katetushkan) [inline index.html]: Same for `<label>` here

File: css/siteStyle.css [hunk 5]
```diff
@@ -0,0 +1,56 @@
+html{
+    height: 100vh;
+}
+header{
+    background: #ffa573;
+    position: fixed;
+    width: 100%;
+    height: 18vh;
```
Comment (katetushkan) [inline css/siteStyle.css]: don't use vh, it could be tricky on moblie devices.

File: css/siteStyle.css [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    background: url("../images/headerr.png") no-repeat;
+    -webkit-background-size: 100%;
+    left: 0;
+    right: 0;
+    top:0;
+}
+body {
+    display: flex;
+    background: #ffa573;
+    padding-top: 18vh;
```
Comment (katetushkan) [inline css/siteStyle.css]: same here. use rem

Comment (katetushkan) [reply css/siteStyle.css]: also it's better to have body with margins and padding 0. Use margins for ur inner components

File: css/StartPlantingPage.css [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+    font-weight: bold;
+    color: #F5E2E2;
+    background: #714444;
+
+    text-align: center;
+    border: none;
+    border-top-left-radius: 20px;
+    border-top-right-radius: 20px;
+}
+aside{
```
Comment (katetushkan) [inline css/StartPlantingPage.css]: use class

File: css/LoggedPage.css [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+    width: 100%;
+    height: 18vh;
+    background: url("../images/headerr.png") no-repeat;
+    -webkit-background-size: 100%;
+    left: 0;
+    right: 0;
+    top:0;
+}
+body {
+    /*display: flex;*/
```
Comment (katetushkan) [inline css/LoggedPage.css]: should be flex if u wanna full screen it

File: css/siteStyle.css [hunk 9]
```diff
@@ -0,0 +1,56 @@
+html{
+    height: 100vh;
```
Comment (katetushkan) [inline css/siteStyle.css]: no. 100%

File: css/SignUpPage.css [hunk 10]
```diff
@@ -0,0 +1,57 @@
+.log-form{
+    position: fixed;
```
Comment (katetushkan) [inline css/SignUpPage.css]: position fix delete elements from the elements flow. if u wanna place it center so this :
1) make container for form flex;
2) set flex-shrink for kids.

File: css/LoggedPage.css [hunk 11]
```diff
@@ -0,0 +1,96 @@
+header{
+    background: #ffa573;
+    position: fixed;
```
Comment (katetushkan) [inline css/LoggedPage.css]: no fixed.

**Review by katetushkan — CHANGES_REQUESTED** [review]
U need to fix problem I mentioned in comments. It doesn't have a mobile version, though. If u wanna more than `8` u need to add it, as well as focus/hover styles to ur buttons.

File: html/StartPlantingPage.html (hunk 1 above)
Comment (NovikVeronika) [reply html/StartPlantingPage.html]: Yes, it should. At first i thought that the start would be a jump to a new page, but now i understand that this is unnecessary. I added all the changes. Now it works correctly in mobile version too. :)

**Review by katetushkan — APPROVED** [review]
`9` is urs.

**Pull request merged**